# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: ipc/glue/IPCMessageUtilsSpecializations.h
# Commit: 55f13fb4ee3f
# Full Hash: 55f13fb4ee3f06209d00fd91a7a03884bab88e2d
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-12-15 09:27:59
# Regressor Bug: 1247687
# File Overlap Count: 1
# Description:
#   Bug 1247687 - Implement csp for Module Workers; r=evilpie,asuth,rpl,ckerschb
#   
#   Depends on D155691
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D156102
# ==============================================================================

diff -r 354711cf113a -r 55f13fb4ee3f ipc/glue/IPCMessageUtilsSpecializations.h
--- a/ipc/glue/IPCMessageUtilsSpecializations.h	Wed Dec 14 14:55:37 2022 +0000
+++ b/ipc/glue/IPCMessageUtilsSpecializations.h	Wed Dec 14 14:55:38 2022 +0000
@@ -328,9 +328,9 @@
 
 template <>
 struct ParamTraits<nsContentPolicyType>
-    : public ContiguousEnumSerializerInclusive<
-          nsContentPolicyType, nsIContentPolicy::TYPE_INVALID,
-          nsIContentPolicy::TYPE_WEB_IDENTITY> {};
+    : public ContiguousEnumSerializer<nsContentPolicyType,
+                                      nsIContentPolicy::TYPE_INVALID,
+                                      nsIContentPolicy::TYPE_END> {};
 
 template <>
 struct ParamTraits<mozilla::TimeDuration> {